Sources/StreamChatUI/Composer/TypingSuggester.swift

```diff
@@ -100,17 +100,10 @@ public struct TypingSuggester {
             return nil
         }
 
-        // Fetch the suggestion text. The suggestions can't have spaces.
```

Comment:
Why do we allow spaces now?

Reply:
It is a bug. If you typed "@" in a new line, it wouldn't show the suggestions

Reply:
It looks this was wrong before, user names can have spaces. The other platforms also allow spaces in user suggestions